versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 2"
```

Comment:
I don't remember changing this.. assuming this was done by automation.

Reply:
Change done by me